### DISCUSSION TITLE
Cannot enter OMDB Api

### DISCUSSION THREAD

**Discussion by suchAflo** [chhoumann/quickadd#429] [Q&A]
hi there hope i am in the right place for this:

i am trying to add this script to quick add/obsidian and follow the minimal tutorial. its not too hard but after i added the script in quick add and click the cog i should be able to enter the api key - but no window opens that lets me do that.. 

any hints und why that happens?

thx so much

**Comment by dang1507** [comment]
I'm also having this issue.

**Comment by chhoumann** [comment]
Can you check the developer console in Obsidian for errors when trying to click the cog?
If there are, please provide a screenshot of the error so I can diagnose. :)

CTRL/Cmd+Shift+i on Windows/Mac.

**Comment by dang1507** [comment]
This is what I am seeing when I try to open the configuration for the API keys

`VM664:8 Uncaught (in promise) SyntaxError: Unexpected token '<'
    at eval (<anonymous>)
    at getUserScript (plugin:quickadd:32257:24)`

![image](https://user-images.githubusercontent.com/60278428/218933913-15220ec2-1f51-4e2d-ab69-e6a74359d9f9.png)

![image](https://user-images.githubusercontent.com/60278428/218935079-4e170e8d-26a9-4907-9f95-191b3294caee.png)

**Comment by chhoumann** [comment]
Thank you, @dang1507!

Can you share the script as well? An image is fine, but preferably copy pasted here.

**Comment by chhoumann** [comment]
Coming back to this one very late - sorry about that.

@dang1507's console error points at the most likely cause: `SyntaxError: Unexpected token '<'` while loading a user script almost always means the `.js` file actually contains an HTML page - typically from saving the GitHub page itself rather than the Raw file, so the file starts with `<html>`/`<!DOCTYPE` and the script loader chokes on the first `<`. When the script can't load, the cog has no settings to show, which matches what you both saw.

The fix: re-download the script from the direct download link on the docs page - https://quickadd.obsidian.guide/docs/Examples/Macro_MovieAndSeriesScript - then open the file in a text editor to confirm it's plain JavaScript, keep the `.js` extension, and store it in your vault outside the `.obsidian` folder. The docs page also has a troubleshooting section for this script now.

@suchAflo if that doesn't sort it (or your case is different), can you share the error from the developer console (Ctrl/Cmd+Shift+I) when you click the cog?
